src/test/src/catalogfunctions-test.cc

```diff
@@ -201,25 +201,25 @@ TEST_F(CatalogFunctionsTest, GetInfoTest){
   EXPECT_STREQ("table", buf);
 }
 
-TEST_F(CatalogFunctionsTest, DISABLED_PrimaryKeysTest){
+TEST_F(CatalogFunctionsTest, PrimaryKeysTest){
   rc_ = SQLPrimaryKeys(hstmt_,
                        NULL, 0,
                        (SQLCHAR *) "postgres.tpch", SQL_NTS,
                        (SQLCHAR *) "lineitem", SQL_NTS);
   CHECK_STMT_RESULT(rc_, "SQLPrimaryKeys failed", hstmt_);
   std::string err_msg;
-  std::string exp_result_meta = "TABLE_CAT: WVARCHAR(1024) digits: 0, nullable\n"
-                                "TABLE_SCHEM: WVARCHAR(1024) digits: 0, nullable\n"
-                                "TABLE_NAME: WVARCHAR(1024) digits: 0, not nullable\n"
-                                "COLUMN_NAME: WVARCHAR(1024) digits: 0, not nullable\n"
-                                "KEY_SEQ: SMALLINT(5) digits: 0, not nullable\n"
-                                "PK_NAME: WVARCHAR(1024) digits: 0, nullable\n";
+  std::string exp_result_meta = "TABLE_CAT: VARCHAR(1024) digits: 0, nullable\n"
```

Comment:
Is there a reason to change WVARCHAR to VARCHAR? Am I right that we do not support WVARCHAR?

Reply:
We do, but it's client specific. For W functions it should be WVARCHAR